tools/cli/admin.go

```diff
@@ -520,6 +524,42 @@ func newAdminElasticSearchCommands() []cli.Command {
 				AdminIndex(c)
 			},
 		},
+		{
+			Name:    "delete",
+			Aliases: []string{"del"},
+			Usage:   "Delete docs on ElasticSearch",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  FlagURL,
+					Usage: "URL of ElasticSearch cluster",
+				},
+				cli.StringFlag{
+					Name:  FlagMuttleyDestinationWithAlias,
+					Usage: "Optional muttely destination to ElasticSearch cluster",
+				},
+				cli.StringFlag{
+					Name:  FlagIndex,
+					Usage: "ElasticSearch target index",
+				},
+				cli.StringFlag{
+					Name:  FlagInputFileWithAlias,
+					Usage: "Input file of indexer.Message in json format, separated by newline",
```

Comment:
Please explain more detail about the file format

Reply:
addressed